build_r_site.R

```diff
@@ -1,20 +0,0 @@
-library(pkgdown)
-library(roxygen2)
-
-setwd("lightgbm_r")
-if (!dir.exists("docs")) {
-  dir.create("docs")
-}
```

Comment:
Why can we drop this?

Reply:
`pkgdown::build_site()`  creates a `doc/` directory automatically: https://github.com/r-lib/pkgdown/blob/aea6864792d681f64a9e1b8c52fe24668b2e4f55/R/build.r#L23-L24